packages/h3xt/index.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const fs = require('fs');
+const http = require('http-next');
+
+module.exports = (opts) => new Application(opts);
+
+class Application extends Function {
+  constructor (opts = {}) {
+    // Application is a valid handler function
+    super('req', 'return this.handle.call(this, req)');
+    this.opts = opts;
+    this.server = null;
+    const self = this.bind(this);
+    self.listen = this.listen.bind(this);
+    return self;
+  }
+
+  listen (...args) {
+    // Create underlying server instance
+    this.server = http.createServer({
+      allowHTTP1: typeof this.opts.allowHTTP1 === 'boolean' ? this.opts.allowHTTP1 : true,
+      allowHTTP2: typeof this.opts.allowHTTP1 === 'boolean' ? this.opts.allowHTTP2 : true,
```

Comment:
```suggestion
      allowHTTP2: typeof this.opts.allowHTTP2 === 'boolean' ? this.opts.allowHTTP2 : true,
```
or was this intentional?